google-cloud-logging/test/google/cloud/logging/project/list_entries_test.rb

```diff
@@ -224,7 +224,7 @@
     list_res = OpenStruct.new(page: OpenStruct.new(response: Google::Logging::V2::ListLogEntriesResponse.decode_json(list_entries_json(3, "next_page_token"))))
 
     mock = Minitest::Mock.new
-    mock.expect :list_log_entries, list_res, [["project1"], resource_names: nil, filter: nil, order_by: nil, page_size: nil, options: default_options]
+    mock.expect :list_log_entries, list_res, [[], project_ids: ["project1"], filter: nil, order_by: nil, page_size: nil, options: default_options]
```

Comment:
I believe the behavior for this test is correct. The way Logging works is if you don't specify the request with a resource or project we will scope it to the project that the client is authenticated with. If the user specifies alternate projects ids to scope the request to then we should not add the current project as a resource.

The API has not changed, only the GAPIC code has. We are simply continuing the behavior that was in place since before resource names were added to the API.